.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "makefile.extensionOutputFolder": "./.vscode"
+}
```

Comment:
Was this folder created automatically? if so, could you deleted it for me?

Reply:
For some reason even added in .gitignore this folder still appears.
Folder deleted.